src/main/kotlin/no/fdk/terms/controller/TermsController.kt

```diff
@@ -55,4 +55,10 @@ class TermsController(
             ?: ResponseEntity(HttpStatus.NOT_FOUND)
     }
 
+    @GetMapping(value = ["/latest/version"], produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
@stigbd Sa egentlig at jeg ikke skulle legge disse til, men orket ikke å bruke mer tid på å få sso-scriptet til å tolke json. Så for at det skal funke må jeg ha dette endepunktet, men kan flytte det til user-api